tests/test_other.py

```diff
@@ -4480,6 +4480,22 @@ def test_syscall_without_filesystem(self):
 }''')
     run_process([PYTHON, EMCC, 'src.c', '-s', 'NO_FILESYSTEM=1'])
 
+  def test_dashS(self):
```

Comment:
Nit: could be `dash_S` to match `has_dash_S`? `test_dashS_stdout` looks like it's mixing camel and snake case to me, the underscore makes that less odd I feel.

Reply:
I'm just mimicking the `test_dashE_xx` and `test_dash_M_xxx` tests below.